packages/auth/README.md

```diff
@@ -332,10 +332,18 @@ import { getCurrentUser, hasRole } from 'src/lib/auth'
 //    return { ...user, roles: roles(decoded) }
 //  }
 //
-//  export const hasRole = (role) => {
-//    if (!context.currentUser.roles?.includes(role)) {
+// Use this function in your services to check that a user is logged in,
+// whether or not they are assigned a role, and optionally raise an
+// error if they're not.
+//
+//  export const requireAuth = ({ role }) => {
+//    if (!context.currentUser) {
 //      throw new AuthenticationError("You don't have permission to do that.")
 //    }
+//
+//    if (!context.currentUser.roles?.includes(role)) {
```

Comment:
```suggestion
//    if (typeof role !== 'undefined' && !context.currentUser.roles?.includes(role)) {
```

Reply:
The specification of a role could be optional.

Reply:
Yup, I noticed that as well and had not yet committed:

`if (role && !context.currentUser.roles?.includes(role)) {`

but will use your suggestion,